corehq/apps/app_manager/models.py

```diff
@@ -1316,6 +1316,28 @@ def _get_active_actions(self, types):
                 actions[action_type] = a
         return actions
 
+    @memoized
+    def get_action_type(self):
+        if self.get_subcase_types():
```

Comment:
It seems weird to me that a method called `get_subcase_types` gets cases opened by the form (as opposed to child cases opened by the form) but it looks like that is indeed the way it is.